src/summary.rs

```diff
@@ -129,6 +129,8 @@ impl Summary {
         {
             msg.get_file(context)
                 .and_then(|path| path.to_str().map(|p| p.to_owned()))
+        } else if msg.viewtype == Viewtype::Webxdc {
+            Some("msg-icon://last-msg-id".to_string())
```

Comment:
The suggestion was "webxdc-icon:last-msg-id", but i think:
- This should be path-like, so let's use "msg-icon://" as a prefix
- Better not to add "webxdc", this way the string can be reused for other viewtypes if we want. It's just a hint, how the icon should be taken exactly is up to the app

Reply:
actually it should contain webxdc IMO, because the ui in desktop just replaces it with the url for the custom uri scheme that serves webxdc icons. I don't see much point in having an api that requires an extra if statement in ui code.

Reply:
I changed it to "webxdc-icon://last-msg-id"